### DISCUSSION TITLE
No response after clicking the build server button

### DISCUSSION THREAD

**Discussion by DOMGO-cut** [postalserver/postal#3060] [Help with using Postal] [answered]
https://****/org/***/servers/new

{"status":500,"error":"Internal Server Error"}
application-0dd293ea0a8f7f37880ae5bea4a4f1627195bd86b17430b7fb0313790f195df8.js

**Comment by DOMGO-cut** [comment]
Loading config from /config/postal.yml
3.3.4

**Comment by willpower232** [comment, marked as answer]
Most of the time this issue is because the second set of database connection details (called message_db in the config file) has not worked.

Make sure that the user in that bit can use `postal_*` databases.
